src/components-examples/cdk/tree/cdk-tree-flat/cdk-tree-flat-example.ts

```diff
@@ -88,7 +88,13 @@ export class CdkTreeFlatExample {
   }
 
   shouldRender(node: ExampleFlatNode) {
-    const parent = this.getParentNode(node);
-    return !parent || parent.isExpanded;
+    let parent = this.getParentNode(node);
+    while (parent) {
+        if (!parent.isExpanded) {
```

Comment:
The indentation is off on the entire block.

Reply:
Changed on commit [06373d3](https://github.com/angular/components/commit/06373d3877ac987ec23da2b38675ea0bbf4d4497)